gradle.properties

```diff
@@ -39,10 +39,10 @@ TARGET_COMPATIBILITY = 1.8
 
 ### Dependencies
 
-LIB_GRAPHQL_JAVA_VER = 14.1
-LIB_SPRING_BOOT_VER = 2.3.1.RELEASE
-LIB_GRAPHQL_SERVLET_VER = 9.2.0
-LIB_GRAPHQL_JAVA_TOOLS_VER = 6.1.0
+LIB_GRAPHQL_JAVA_VER = 15.0
+LIB_SPRING_BOOT_VER = 2.3.4.RELEASE
+LIB_GRAPHQL_SERVLET_VER = 9.3.0-SNAPSHOT
```

Comment:
Maybe it is 9.2.0-SNAPSHOT ?

Reply:
graphql-java-servlet 9.2.0 is already released - https://github.com/graphql-java-kickstart/graphql-java-servlet/releases

The next working version would be 9.3.0-SNAPSHOT

Reply:
Has now been merged and released at graphql-java-servlet 10.0.0

https://github.com/graphql-java-kickstart/graphql-java-servlet/releases/tag/v10.0.0